Reference/conformance/conformance.pb.swift

```diff
@@ -230,11 +230,28 @@ struct Conformance_ConformanceResponse: SwiftProtobuf.Message {
   var unknownFields = SwiftProtobuf.UnknownStorage()
 
   enum OneOf_Result: Equatable {
```

Comment:
Repeated the oneof comment on the enum also.  Hopefully this will let Xcode's help/tips provide some info in all cases.